src/main/resources/db/changelog/20220329154201-owner-product-key-change.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="20220329154201-1" author="wpoteat">
+        <addColumn tableName="cp2_owner_products">
+            <column name="product_id" type="VARCHAR(255)"/>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="20220329154201-2" author="wpoteat" dbms="postgresql">
+        <sql>
+            UPDATE cp2_owner_products op
+            SET product_id = prod.product_id
+                FROM cp2_products prod
+                WHERE op.product_uuid = prod.uuid
+        </sql>
+    </changeSet>
+
+    <changeSet id="20220329154201-3" author="wpoteat"  dbms="mysql, mariadb">
+        <sql>
+            UPDATE cp2_owner_products op
+            SET product_id=(SELECT prod.product_id
```

Comment:
We should use the UPDATE-JOIN syntax here rather than a subquery:
```
UPDATE cp2_owner_products op JOIN cp2_products p ON p.uuid = op.product_uuid SET op.product_id = p.product_id
```